pom.xml

```diff
@@ -1578,7 +1591,7 @@ under the License.
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.22.1</version>
+				<version>2.22.2</version>
```

Comment:
Why is this being bumped?

Reply:
Thanks for the review @zentol! 2.22.2 [fixed a bug](https://issues.apache.org/jira/browse/SUREFIRE-1614) related to junit-vintage-engine, so I bumped the version.